core/src/main/java/org/apache/calcite/util/ImmutableBitSet.java

```diff
@@ -901,6 +902,19 @@ public ImmutableBitSet shift(int offset) {
     return builder.build();
   }
 
+  /**
+   * Checks if all bit sets contain a particular bit.
+   */
+  public static boolean allContain(Collection<ImmutableBitSet> bitSets,
+                                   int bit) {
```

Comment:
we do not format like this

Reply:
Revised. Thank you for your kind reminder.